examples/framework-bun/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "node",
```

Comment:
By default this is `"bundler"` which makes all of the exports defined in the `./inngest` files have this TS error:
```
The inferred type of 'functions' cannot be named without a reference to '../node_modules/inngest/types'. This is likely not portable. A type annotation is necessary.ts(2742)
```

Reply:
I've moved it back to the default of `"bundler"` - this is a good test for issues like #384 (also see #385 / #460).